variants/heltec_hru_3601/platformio.ini

```diff
@@ -0,0 +1,8 @@
+[env:heltec-hru-3601]
+extends = esp32c3_base
+board = adafruit_qtpy_esp32c3
+board_level = extra
+build_flags = 
+  ${esp32_base.build_flags}
+  -D PRIVATE_HW
```

Comment:
If you have a on-sale device i'd recommend registering a public enumerator in the protobuf registry for it

Reply:
I'm not actually working for Heltec but just a random user of this device, but I have opened a PR in protobufs for it: https://github.com/meshtastic/protobufs/pull/521